ceph-dashboard-cephadm-e2e/config/definitions/ceph-dashboard-cephadm-e2e.yml

```diff
@@ -0,0 +1,84 @@
+- job:
+    name: ceph-dashboard-cephadm-e2e
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    node: huge && focal && x86_64
+    display-name: 'ceph: Dashboard + Cephadm E2E'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 15
+          num-to-keep: 300
+          artifact-days-to-keep: -1
+          artifact-num-to-keep: -1
+      - github:
+          url: https://github.com/ceph/ceph/
+      - rebuild:
+          auto-rebuild: true
+      - inject:
+          properties-content: |
+            TERM=xterm
+
+    parameters:
+      - string:
+          name: sha1
+          description: "commit id or a refname, like 'origin/pr/72/head'"
+
+    triggers:
+      - github-pull-request:
+          cancel-builds-on-update: true
+          allow-whitelist-orgs-as-admins: true
+          org-list:
+            - ceph
+          white-list-labels:
+#            - dashboard
+            - wip-alfonsomthd-testing
+          black-list-target-branches:
+            - luminous
+            - mimic
+            - nautilus
+            - octopus
```

Comment:
Shouldn't we run this on octopus ones too?

Reply:
As per our previous conversation will run it in octopus but take into account that the octopus ceph backport will be needed to succeed.